PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,13 +9,13 @@
 - [ ] All changes relevant to installation and setup are reflected in `DEVELOPMENT.md` and `PRODUCTION.md`
 - [ ] The changes have passed user-acceptance testing (if required by the originating issue)
 
-### Breaking changes
+### :axe: Breaking changes
 
 >[!WARNING]
 >
```

Comment:
```
>[!WARNING]
> None
```
please change also this for the correct display of this Markdown

Reply:
I don't like the idea of having any default value. I also think that if there aren't any breaking changes the whole block should be removed. It doesn't make sense for me having a [!WARNING] block that doesn't contain any information. 

It will also catch the eye if we only have the block on PR that indeed have breaking changes on it. What do you think?